src/isDeepEqual.ts

```diff
@@ -1,11 +1,11 @@
 import { purry } from "./purry";
 
 /**
- * Performs a deep *semantic* comparison between two values to determine if they
- * are equivalent. For primitive values this is equivalent to `===`, for arrays
- * the check would be performed on every item recursively, in order, and for
- * objects all props will be compared recursively. The built-in Date and RegExp
- * are special-cased and will be compared by their values.
+ * Performs a _deep_ *semantic* comparison between two values to determine if
```

Comment:
**nit:** both `_` and `*` are italics in markdown, so maybe this should be `*deep semantic*`?

i think we can also drop the word "semantic" here. i don't think it's helpful

Reply:
if we want to keep a word like semantic, mahybe "structural" would be it, but idk, i think "deep" alone conveys what's needed?

Reply:
I agree, it was confusing to me too